### PR TITLE
[CLOUDTRUST-2511] Replace SetGroupsToUser by Add/Delete methods

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

File: pkg/management/authorization.go [hunk 1]
```diff
@@ -265,13 +265,27 @@ func (c *authorizationComponentMW) SetGroupsToUser(ctx context.Context, realmNam
...
+
+func (c *authorizationComponentMW) DeleteGroupForUser(ctx context.Context, realmName, userID string, groupID string) error {
+	var action = MGMTSetGroupsToUser.String()
+	var targetRealm = realmName
+
+	if err := c.authManager.CheckAuthorizationOnTargetUser(ctx, action, targetRealm, userID); err != nil {
+		return err
+	}
+
+	action = MGMTAssignableGroupsToUser.String()
```
Comment (bsoniam) [inline pkg/management/authorization.go:283]: same action as when we add ?

File: cmd/keycloakb/keycloak_bridge.go [hunk 2]
```diff
@@ -907,7 +909,8 @@ func main() {
 		managementSubroute.Path("/realms/{realm}/users/{userID}").Methods("PUT").Handler(updateUserHandler)
 		managementSubroute.Path("/realms/{realm}/users/{userID}").Methods("DELETE").Handler(deleteUserHandler)
 		managementSubroute.Path("/realms/{realm}/users/{userID}/groups").Methods("GET").Handler(getGroupsForUserHandler)
-		managementSubroute.Path("/realms/{realm}/users/{userID}/groups").Methods("PUT").Handler(setGroupsToUserHandler)
```
Comment (bsoniam) [inline cmd/keycloakb/keycloak_bridge.go:910]: setGroupsToUser is not needed anymore?

Comment (fperot74) [reply cmd/keycloakb/keycloak_bridge.go:910]: Yes... it is replaced by Add/Delete methods

File: pkg/management/authorization.go (hunk 1 above)
Comment (fperot74) [reply pkg/management/authorization.go:283]: Add/Delete methods have MGMTSetGroupsToUser rights to be able to update groups of user and have MGMTAssignableGroupsToUser to know which groups can be added/removed

**Comment by fperot74** [comment]
> Please update the swagger as well.

Done

**Pull request merged**